pysalesforce.py

```diff
@@ -1044,6 +1049,90 @@ def query(query_string, access_token, instance_url):
 
         return json_response
 
+    @staticmethod
+    def reset_users_password(user_id, access_token, instance_url):
+        """
+        Resets a users' password.
+
+        Args:
+            user_id (str):      The user id of the user who's password should be reset
+            access_token (str): This is the access_token value received from the
+                                login response
+            instance_url (str): This is the instance_url value received from the
+                                login response
+
+        Returns:
+            object: returns information about the password reset
+        """
+        query_uri = '/sobjects/User/%s/password' % user_id
```

Comment:
I think this should use the newer f string or .format. Everything else in here is using .format, but I'd be fine with f string because I'll probably go through at some point and update everything to that.